README.md

```diff
@@ -111,6 +130,9 @@ interface ConnectionReaddirComplexResult
     size : number
     href : string
     name : string
+    extraProperties: {
+        [name : string] : string | number | boolean
```

Comment:
What are valid `extraProperties` and where can I find a list/description of them?

Reply:
No guarantee for extraProperties, depends on context.